kratos/future/processes/future_process.cpp

```diff
@@ -0,0 +1,32 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Carlos A. Roig
+//
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "future/processes/future_process.h"
+
+namespace Kratos::Future
+{
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+void Process::Execute()
+{
+    KRATOS_ERROR << "I come from the future :)" << std::endl;
```

Comment:
```suggestion
    KRATOS_ERROR << "I come from the future, I wring Neutrex© :)" << std::endl;
```

Reply:
I wanted to add this but we cannot use trademarks :disappointed:

Reply:
Now?